recipes/newrelic/apm/java/rhel.yml

```diff
@@ -0,0 +1,66 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: java-agent-installer
+displayName: Java Agent Installer
+description: New Relic install recipe for instrumenting Java applications
+repository: https://github.com/newrelic/newrelic-java-agent
+
+installTargets:
+  - type: host
+    os: linux
+    platform: amazon
+    platformVersion: "2"
+  - type: host
+    os: linux
+    platform: "centos"
+  - type: host
+    os: linux
+    platform: "redhat"
```

Comment:
This looks like a mistake using `type: host`. Think it should be `type: application` like in the Debian recipe.

Reply:
I made the changes you asked.  I also update the Debian recipe to have definition that are Debian and not redhat.

https://github.com/newrelic/open-install-library/pull/252/commits/161e2e69817d80bd0b65195e427e9756a94945ad